Woof/Woof/ViewModel/DetailPetSitterViewModel.swift

```diff
@@ -9,7 +9,7 @@ final class DetailSitterViewModel: ObservableObject {
     @Published var bio: String
 
     /// The rating of the sitter, on a scale from 0 to 5.
-    @Published var rating: Int
+    @Published var rating: Rating
```

Comment:
I think we should update the documentation here as well.

Reply:
Fixed.